src/clipboard.ts

```diff
@@ -72,17 +73,34 @@ export const copyToClipboard = async (
         }, {} as BinaryFiles)
       : undefined,
   };
-  const json = JSON.stringify(contents);
-  CLIPBOARD = json;
+
+  const clipboardDataAsText = transformClipboardElementsToText(
+    contents.elements,
+  );
+  CLIPBOARD = clipboardDataAsText;
+
   try {
     PREFER_APP_CLIPBOARD = false;
-    await copyTextToSystemClipboard(json);
+    await copyTextToSystemClipboard(clipboardDataAsText);
   } catch (error: any) {
     PREFER_APP_CLIPBOARD = true;
     console.error(error);
   }
 };
 
+export const transformClipboardElementsToText = (
+  elements: readonly NonDeletedExcalidrawElement[],
+): string => {
+  const onlyTextElements = elements.every((element) => element.type === "text");
```

Comment:
```suggestion
  const onlyTextElements = elements.some((element) => element.type === "text");
```

Let's use some as we'll still be able to paste something reasonable as per my suggestion https://github.com/excalidraw/excalidraw/issues/5937#issuecomment-1341640665

Reply:
Ah, I misunderstood your comment, sounds good to me :)